packages/theme/src/theme/_variables.scss

```diff
@@ -77,7 +77,7 @@ $form-input-font-size: 16px !default;
 
 //** By default, this inherits from the `<body>`.
 $headings-font-family: inherit !default;
-$headings-font-weight: 500 !default;
+$headings-font-weight: 600 !default;
 $headings-line-height: 1.1 !default;
 $headings-color: inherit !default;
```

Comment:
You can set it to `$black` here;
And remove all stuff from _type.scss file except h1 font-weight

Reply:
When I search in our theme code, there is no use except the variables file.
When I search in our bootstrap code, there is only headings that use it.

Reply:
and so ? Do you expect something from me ?

Reply:
and so ? Do you expect something from me ?

Reply:
To set it to `$black` and remove stuff in `_types.scss` (see first comment by @frassinier).
(This is related to my other comment)